index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Promise: Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- first-section -->
+  <div class="wrapper">
+      <div class="header-wrapper">
+        <!-- navbar-section -->
+        <header class="header-section">
+          <span class="nav-span"><a href="#">Promise Okechukwu</a></span>
+          <span><a href="#"><img src="./images/Enabled.png" class="#" alt="#"></a></span>
+         
+          <ul class="nav-links">
+            <li>
+              <span><a href="#">Portfolio</a></span>
+              <span><a href="#">About</a></span>
+              <span><a href="#">contact</a></span>
+              <span><a href="#"><img src="./images/mail.png" alt="mail logo"></a></span>
+            </li>
+          </ul>
+        </header>
+        <!-- header  section -->
+        <section class="first-section">
+          <div class="header-intro">
+            <h1 class="header-text">
+              Hey There.
+              I’m Promise
+            </h1>
+            <h2 class="header-smaller-text"> I am a Software Developer </h2>
+            <p class="header-paragraph">
+              I can help you build a product,
+              feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don’t hestiate to contact me.
+            </p>
+          </div>
+          <!-- Icons section -->
+          <div class="header-icons">
+            <ul>
+              <li><a href="https://jheart-vic.github.io/My-Portfolio-Website/"><img src="images/github.png"
+                    alt="Github logo"></a></li>
```

Comment:
[Optional]

- [ ] Your icons are not visible. Please add **./** in the link before the `images/github,png`. :+1: Then it would be visible :muscle: